pytorch_toolkit/ote/ote/misc.py

```diff
@@ -115,3 +117,33 @@ def run_with_termination(cmd):
 def get_work_dir(cfg, update_config):
     overridden_work_dir = update_config.get('work_dir', None)
     return overridden_work_dir[0][1] if overridden_work_dir else cfg.work_dir
+
+
+def download_snapshot_if_not_yet(template_file, output_folder):
```

Comment:
May be it's better to make this function common for `instantiate_template.py` too?

Reply:
made